app/routes/docs.client.samples.md

```diff
@@ -164,3 +164,61 @@ for message in consumer.consume(end[0].offset - start[0].offset, timeout=1):
             continue
     print(message.value())
 ```
+
+## Working With JSON Schema
+
+For new missions, GCN Notices are preferably distributed in JSON format. This guide describes how to programmatically read the JSON schema.
+
+## Parsing JSON
+
+Read the JSON data from [sample.schema.json](https://gcn.nasa.gov/docs/notices/schema) and [sample.example.json](https://gcn.nasa.gov/docs/notices/schema), which parses it into Python dictionaries.
+
+```python
+from gcn_kafka import Consumer
+import json
+
+# Connect as a consumer
+consumer = Consumer(client_id='fill me in',
+                    client_secret='fill me in')
+
+# Subscribe to topics and receive alerts
+consumer.subscribe(['gcn.classic.voevent.FERMI_GBM_SUBTHRESH'])
+
+# Continuously consume and parse messages as JSON
+for message in consumer.consume(timeout=1):
+    if not message.error():
+        json_data = json.loads(message.value().decode('utf-8'))
+        print("Received JSON Notice:", json_data)
+```
+
+## Encoding and Decoding of Embedded Data
+
+The following code sample shows how to encode/decode a file in Python. The [`base64`](https://docs.python.org/3/library/base64.html#base64.b64encode) package includes the methods `b64decode` and `b64encode` to make this task simple.
```

Comment:
> The following code sample shows how to encode/decode a file in Python.

It doesn't show how to encode/decode _any_ file in Python; only how to encode/decode bytes to base64 for transfer over an ASCII medium.

Reply:
```suggestion
```python
import base64

# Encode and parse the content of your file to a base64 encoded string
with open("path/to/your/file", 'rb') as file:
    encoded_string = base64.b64encode(file.read())

print(encoded_string)
# b'a1512dabc1b6adb3cd1b6dcb6d4c6......'

with open("path/to/encoded_file.txt", 'wb') as encoded_file:
    encoded_file.write(encoded_string)

# Decode and write the content to a local file:
with open("path/to/encoded_file.txt", 'rb') as encoded_file:
    decoded_data = base64.b64decode(encoded_file.read())

with open("path/to/destination/file", 'wb') as file:
    file.write(decoded_data)
```
```

Reply:
```suggestion
The following code sample shows how to encode/decode a file in Python. The [`base64`](https://docs.python.org/3/library/base64.html#base64.b64encode) package includes the methods `b64decode` and `b64encode` to make this task simple.

```python
import base64
# Encode and parse the content of your file to a base64 encoded string
with open("path/to/your/file", 'rb') as file:
    encoded_string = base64.b64encode(file.read())
print(encoded_string)
# b'a1512dabc1b6adb3cd1b6dcb6d4c6......'
with open("path/to/encoded_file.txt", 'wb') as encoded_file:
    encoded_file.write(encoded_string)
# Decode and write the content to a local file:
with open("path/to/encoded_file.txt", 'rb') as encoded_file:
    decoded_data = base64.b64decode(encoded_file.read())
with open("path/to/destination/file", 'wb') as file:
    file.write(decoded_data)
```

@dakota002 could you please help for details?